### ISSUE TITLE
As a project applicant I want to know if my application is still pending 

### ISSUE THREAD

**Issue by tsuhachev** [zerocracy/farm#1776]
Hi, I applied for JPeek (this is the only available Java sandbox project for now) project on 23.10.18

I know that according to the policy  "The architect of the project will either invite you or ignore your request", but it would be clearer for me as an applicant to receive at least some acknowledgement status that my application was actually reviewed, but ignored.

Otherwise, I do not know the review process is finished or in a pending state.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@tsuhachev[/z](https://www.0crat.com/u/tsuhachev) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by tsuhachev** [comment]
@g4s8 , I don't think so, even though the case is the same, the current ticket is about letting know an applicant, not an ARC or PO

**Comment by g4s8** [comment]
@tsuhachev let's implement #1349 first, and use `applications.xml` for this task also.

**Comment by 0crat** [comment]
Job [#1776](https://github.com/zerocracy/farm/issues/1776) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @tsuhachev[/z](https://www.0crat.com/u/tsuhachev)
